### PR TITLE
List systems from db and return via API

### PR DESCRIPTION
**Pull request by Josca**
SPM-27

### CODE REVIEW

File: manager/controllers/paging.go [hunk 1]
```diff
@@ -0,0 +1,66 @@
+package controllers
+
+import "fmt"
+
+func CreateLinks(path string, offset, limit, total int, dataFormat string, showAll bool) Links {
```
Comment (semtexzv) [inline manager/controllers/paging.go]: What's the reason for show_all parameter ?

Comment (Josca) [reply manager/controllers/paging.go]: to show all systems or only for current account

Comment (jiridostal) [reply manager/controllers/paging.go]: It should allow you to show all Advisories, not only those that are affecting your environment. Currently it's not a requirement so we can remove it.

**Comment by Josca** [comment]
@jiridostal @semtexzv I've removed `show_all` param and added generic `otherParams` param to enable inserting of additional query params co `CreateLinks` method.

**Pull request merged**